tests/manage/mcg/test_bucket_deletion.py

```diff
@@ -4,31 +4,50 @@
 
 from ocs_ci.framework import config
 from ocs_ci.framework.pytest_customization.marks import tier1
+from ocs_ci.utility.utils import check_if_executable_in_path
 
 logger = logging.getLogger(__name__)
 
 
-@pytest.mark.skipif(condition=True, reason="MCG is not deployed")
 @pytest.mark.filterwarnings(
     'ignore::urllib3.exceptions.InsecureRequestWarning'
 )
+@pytest.mark.skipif(
+    condition=config.ENV_DATA['platform'] != 'AWS',
+    reason="Tests are not running on AWS deployed cluster"
+)
 @tier1
 class TestBucketDeletion:
     """
     Test bucket Creation Deletion of buckets
     """
-
-    @pytest.mark.skipif(
-        condition=config.ENV_DATA['platform'] != 'AWS',
-        reason="Tests are not running on AWS deployed cluster"
-    )
+    # TODO: remove skipif
     @pytest.mark.polarion_id("OCS-1299")
     def test_s3_bucket_delete(self, mcg_obj, bucket_factory):
         """
         Test deletion of bucket using the S3 SDK
         """
+        for bucketname in bucket_factory(3, 'S3'):
+            logger.info(f"Deleting bucket: {bucketname}")
+            mcg_obj.s3_delete_bucket(bucketname)
+            assert mcg_obj.s3_verify_bucket_exists(bucketname) is False, \
```

Comment:
Please don't break lines with '\'
Also, use 'assert not..' instead of 'assert... is False'
In this case, you can do:
```python
assert not mcg_obj.s3_verify_bucket_exists(bucketname), (
    f"Found {bucketname} that should've been removed"
)
```